src/common/dom/apply_themes_on_element.ts

```diff
@@ -1,26 +1,12 @@
-import { derivedStyles } from "../../resources/styles";
+import { derivedStyles, darkStyles } from "../../resources/styles";
 import { HomeAssistant, Theme } from "../../types";
+import Color from "color";
```

Comment:
This package is 22kb https://bundlephobia.com/result?p=color@3.1.2

That's the problem with classes, you need to pull it all in. Can you try to find a package that just exports functions so we only pull in the code that we need?

Reply:
Especially because this code is in the main bundle.

Reply:
Yeah, was hard to find a good small one, will search some more